Marlin/ubl_G29.cpp

```diff
@@ -385,7 +385,20 @@
       //  return;
       //}
     }
-    //*/
+    */
+
+    if (code_seen('G')) {
```

Comment:
We can use G.   Both PronterFace and Repetier now handle it just fine.   And of course, SD-Memory cards don't care.